include/actionlib/client/action_client.h

```diff
@@ -207,18 +207,26 @@ class ActionClient
 
   void initClient(ros::CallbackQueueInterface* queue)
   {
-    status_sub_   = queue_subscribe("status",   1, &ActionClientT::statusCb,   this, queue);
-    feedback_sub_ = queue_subscribe("feedback", 1, &ActionClientT::feedbackCb, this, queue);
-    result_sub_   = queue_subscribe("result",   1, &ActionClientT::resultCb,   this, queue);
+    // read parameters indicating publish/subscribe queue sizes
+    int pub_queue_size;
+    int sub_queue_size;
+    n_.param("actionlib_client_pub_queue_size", pub_queue_size, 1);
+    n_.param("actionlib_client_sub_queue_size", sub_queue_size, 10);
```

Comment:
I made the changes and did a rebase, but it's not currently showing up on Github. I cloned the repository to a new directory to test, and my changes show up there. I'll let Github's front end have a few hours to register the change.